keyboards/edgepromx/info.json

```diff
@@ -0,0 +1,64 @@
+{
+  "keyboard_name": "EdgeProMX",
+  "url": "https://github.com/edgepro-dev",
+  "maintainer": "jturnsek",
+  "manufacturer": "EdgePro",
+  "usb": {
+    "vid": "0x1FC9",
+    "pid": "0x0135",
```

Comment:
This appears to be NXP's Vendor ID, and shouldn't be used here. 
https://www.the-sz.com/products/usbid/index.php?v=0x1FC9&p=&n=

Reply:
I will try to submit for the NXP customer free PID program.

Reply:
I have submit PR to pid.codes for VID:0x1209 and PID:4C58. Although hasn't been approved yet, I went ahead and change existing NXP VID:PID. But in parallel I have also requested the free PID form NXP, but I don't know how it will turn. Lets cross fingers:)

Reply:
Regarding the changes in config.h, I do not know why you suggested to remove those lines, which I see in lots of other keyboards implementation. Where do you suggest to move them?